### ISSUE TITLE
Help with applying certain themes

### ISSUE THREAD

**Issue by daleao** [darkthemer/rosepine-theme-windows#2]
Hi there! I tried following all the included readme's but I'm having trouble reproducing the same effect as in some of your screenshots, and I was hoping you could help me out.

I'm on Windows 11 Pro 24H2.
1. **Explorer:** I applied niivu's theme and installed Explorer Patcher, but I couldn't get the ribbon and titlebar to look like yours. Could you perhaps share your Explorer Patcher settings? Or any additional tweaks required?

What it currently looks like:
![s5tvwNi](https://github.com/user-attachments/assets/2ea10277-3a76-4591-b8de-295dd3b656fe)

What it should look like:
![s5tvwNi](https://github.com/user-attachments/assets/b0e826f2-34d0-462b-8af6-6a97a492c560)

Basically the titlebar is black instead of Rose Pine, and still displays the actual title. I also couldn't get the Recycle Bin showing without pinning to Quick Access. How do you achieve that?

2. **Steam:** The Millennium theme is applied but the top right buttons (minimize, maximize, close) are not changed:

What it currently looks like:
![sv3nGNl - Imgur](https://github.com/user-attachments/assets/49149c3a-98f2-42bc-aea5-cd689b8b92d5)

What is should look like:
![7hyv6o5 - Imgur](https://github.com/user-attachments/assets/f16d203c-3c88-40a4-a76b-572ebde1d03d)

Appreciate if you could help me get these right. Amazing theme collection btw!

**Comment by darkthemer** [comment]
Hello!

> 1. **Explorer:** I applied niivu's theme and installed Explorer Patcher, but I couldn't get the ribbon and titlebar to look like yours. Could you perhaps share your Explorer Patcher settings? Or any additional tweaks required?

I didn't use ExplorerPatcher for this. What you need is MicaForEveryone (to fix titlebar color) and StartAllBack (to hide ribbon and titlebar name). You might want to check out Windhawk if you don't want to pay for StartAllBack. Although, I haven't tried this software yet. As for the pinned items in the navigation panel, this requires Winaero Tweaker, you can use my config found in the `_cfg` folder.

> 2. **Steam:** The Millennium theme is applied but the top right buttons (minimize, maximize, close) are not changed:

To fix that, you must copy the `mods.css` contents into the skin folder's `Steam.css` as stated in this repo's `steam/README.md`.

**Comment by daleao** [comment]
Thanks for you reply!

I had MicaForEveryone installed, but the explorer.exe settings were overriding my global settings.
Regarding Steam, I had used the @import method but my syntax was incorrect.
Thanks to your response I was able to fix these two issues.

**Edit:**
If you don't mind, I'm now facing similar issue to Steam but with Discord and Spotify apps.
Using BetterDiscord, I applied the theme in the readme but the buttons didn't change. Is there an additional step to get those?

Similarly for Spicetify, I see there's a custom javascript file provided that should take care of the buttons, but I'm not sure where to place it. I tried placing it inside the text theme folder, but I guess it's not being invoked by anything yet.

Thanks!

**Comment by darkthemer** [comment]
> If you don't mind, I'm now facing similar issue to Steam but with Discord and Spotify apps.
> Using BetterDiscord, I applied the theme in the readme but the buttons didn't change. Is there an additional step to get those?

You would need to use Vencord as it has a settings toggle to use the native titlebar, not BetterDiscord.

> Similarly for Spicetify, I see there's a custom javascript file provided that should take care of the buttons, but I'm not sure where to place it. I tried placing it inside the text theme folder, but I guess it's not being invoked by anything yet.

Refer to the Spicetify docs: https://spicetify.app/docs/advanced-usage/extensions/

**Comment by daleao** [comment]
Vencecord did the trick!
For Spicetify, after installing the extension I end up with this:
![{E894F791-0F77-4938-9AD1-B8A3A0D7DCD4}](https://github.com/user-attachments/assets/4c58cf7d-901c-403e-bf61-b63b26255759)

Can't get rid of the default buttons, and the Rose Pine buttons are a bit larger than other windows.
I did try launching `SpotifyNoControl.exe` from _Extra folder, as well as `--transparent-window-controls` parameter. Neither seem to change anything.
AHK version also doesn't seem to do anything.

**Comment by darkthemer** [comment]
i transfered the rose pine style to the dedicated extension which should now allow you to resize it
https://github.com/darkthemer/spicetify-extensions/tree/main/customControls

also, from now on, i intend to use this windhawk mod instead of the spicetify extension that i made
https://windhawk.net/mods/cef-titlebar-enabler-universal

**Comment by daleao** [comment]
Happy new year, and sorry to keep bothering you.

The Windhawk mod for Spotify worked perfectly.
I reapplied all themes today after a full Windows reinstall.
Was about to close the issue, but decided to try SumatraPDF and MPC-HC, and I'm having issues with those.

SumatraPDF theme doesn't apply to the Menu Bar and Scroll Bar:
![{5CBB6F96-1380-4B26-9A3B-599C52A2FCB5}](https://github.com/user-attachments/assets/dc88f066-8e04-43f8-9bff-a4ceeda09c48)

For MPC I have no idea what to do with the included image files or how to theme the rest of the UI.

**Comment by darkthemer** [comment]
Not sure with SumatraPDF, I updated it to the latest version (from 3.3.3 to 3.5.2) and the menubar and scrollbar on my end is still themed. Try reapplying the Windows theme, maybe?

In MPC-HC, uncheck 'Use Modern Theme' in Options > User Interface. For the custom toolbar: https://superuser.com/questions/499310/how-to-change-toolbar-skin-in-media-player-classic-home-cinema. Basically, just paste `toolbar.png` in the program's folder.
